alta-back-end/src/main/java/com/alta/entity/Task.java

```diff
@@ -49,9 +49,13 @@ public class Task {
     @ManyToOne
     private Topic topic;
 
+//    @ManyToMany(mappedBy = "tasks")
+//    @JsonIgnoreProperties("tasks")
+//    private Set<Student> students = new HashSet<>();
+
     @ManyToMany(mappedBy = "tasks")
     @JsonIgnoreProperties("tasks")
-    private Set<Student> students = new HashSet<>();
+    private Set<TasksGroup> tasks_groups = new HashSet<>();
```

Comment:
_ ?

Reply:
Well, I wanna delete relationship @ManyToMany between Task and Student, create @ManyToMany between Task and TasksGroup.
Am I wrong?